pkg/build/metrics/prometheus/metrics.go

```diff
@@ -93,55 +81,60 @@ func (bc *buildCollector) Collect(ch chan<- prometheus.Metric) {
+		runningCount = runningCount + rc
 	}
 	// explicitly note there are no failed builds
 	if len(reasons) == 0 {
-		addCountGauge(ch, failedBuildCountDesc, "", float64(0))
+		addCountGauge(ch, buildCountDesc, failedPhase, "", float64(0))
 	}
 	for reason, count := range reasons {
-		addCountGauge(ch, failedBuildCountDesc, reason, float64(count))
+		addCountGauge(ch, buildCountDesc, failedPhase, reason, float64(count))
 	}
-	addCountGauge(ch, terminalBuildCountDesc, cancelledPhase, float64(cancelledCount))
-	addCountGauge(ch, terminalBuildCountDesc, completePhase, float64(completeCount))
+	addCountGauge(ch, buildCountDesc, cancelledPhase, "", float64(cancelledCount))
+	addCountGauge(ch, buildCountDesc, completePhase, "", float64(completeCount))
+	addCountGauge(ch, buildCountDesc, newPhase, "", float64(newCount))
 }
 
-func addCountGauge(ch chan<- prometheus.Metric, desc *prometheus.Desc, label string, v float64) {
-	lv := []string{label}
+func addCountGauge(ch chan<- prometheus.Metric, desc *prometheus.Desc, phase, reason string, v float64) {
+	lv := []string{phase, reason}
 	ch <- prometheus.MustNewConstMetric(desc, prometheus.GaugeValue, v, lv...)
 }
 
 func addTimeGauge(ch chan<- prometheus.Metric, b *buildapi.Build, time *metav1.Time, desc *prometheus.Desc, phase string) {
 	if time != nil {
-		lv := []string{b.ObjectMeta.Namespace, b.ObjectMeta.Name}
-		if len(phase) > 0 {
-			lv = append(lv, strings.ToLower(phase))
-		}
+		lv := []string{b.ObjectMeta.Namespace, b.ObjectMeta.Name, phase}
 		ch <- prometheus.MustNewConstMetric(desc, prometheus.GaugeValue, float64(time.Unix()), lv...)
 	}
 }
 
-func (bc *buildCollector) collectBuild(ch chan<- prometheus.Metric, b *buildapi.Build) (cancelledCount, completeCount int, reasonsCount map[string]int) {
+func (bc *buildCollector) collectBuild(ch chan<- prometheus.Metric, b *buildapi.Build) (cancelledCount, completeCount, newCount, pendingCount, runningCount int, reasonsCount map[string]int) {
 
 	reasonsCount = map[string]int{}
 	switch b.Status.Phase {
 	// remember, new and pending builds don't have a start time
 	case buildapi.BuildPhaseNew:
+		newCount++
+		addTimeGauge(ch, b, &b.CreationTimestamp, activeBuildDesc, newPhase)
 	case buildapi.BuildPhasePending:
-		addTimeGauge(ch, b, &b.CreationTimestamp, newPendingBuildCountDesc, string(b.Status.Phase))
+		pendingCount++
+		addTimeGauge(ch, b, &b.CreationTimestamp, activeBuildDesc, pendingPhase)
 	case buildapi.BuildPhaseRunning:
-		addTimeGauge(ch, b, b.Status.StartTimestamp, activeBuildCountDesc, "")
+		runningCount++
+		addTimeGauge(ch, b, b.Status.StartTimestamp, activeBuildDesc, runningPhase)
 	case buildapi.BuildPhaseFailed:
 		// currently only failed builds have reasons
-		reasonsCount[string(b.Status.Reason)] = 1
+		reasonsCount[strings.ToLower(string(b.Status.Reason))] = 1
 	case buildapi.BuildPhaseError:
 		// it was decided to couple this one under failed, using the custom 'BuildPodError'
 		reasonsCount[errorBuildReason] = 1
```

Comment:
now that we're explicitly breaking everything else out, I think it makes sense to revisit this and also break out "error" as its own phase (w/ no reason).

Alerts/queries for "Things are going wrong" can simply query on buildcount for builds in Error+Failed phases.

Reply:
ok yeah i had left the two combined based on earlier iteration feedback but can separate them back out if that is no desired